### PR TITLE
Home Assistant core and frontend update to 2022.05.X, plus many new dependencies

### PR DESCRIPTION
**Pull request by Stefan-Schmidt**
I consider this still a draft PR. Let me know if thsi is the direction you expect this to go.

With the given set of recipes I am able to include HA into and image. It starts up as systemd service and I can access the web interface and go through the onboarding steps to reach the dashboard.

Pretty sure many things are still broken, but its a good start.

Things I still consider a TODO here:
- Decide if we want to keep systemd-notify support or not
- Decide if we want to keep initscript support next to systemd
- Check if all the new dependencies are needed for the core functionality and if we want to bring them in
- My newly added depenencies do not make use of the RDEPENDS handling

https://github.com/meta-homeassistant/meta-homeassistant/issues/59

### CODE REVIEW

**Comment by bachp** [comment]
@Stefan-Schmidt Thanks for the MR it is already.

Some remarks:

- Both systemd-notify and the sysv init script can be dropped. If somebody really needs one of them we can re add it later.
- Did you package all dependencies? The main issue I see is that this is a lot of work. Unless we can [automate it somehow like it's done in nixpkgs](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/servers/home-assistant/update.sh).
- What do you mean by "My newly added depenencies do not make use of the RDEPENDS handling"?

**Comment by Stefan-Schmidt** [comment]
>     * Both systemd-notify and the sysv init script can be dropped. If somebody really needs one of them we can re add it later.

I will drop it from the new 2022.05.x recipe, but leave it for the old one to cause no regression.

>     * Did you package all dependencies? The main issue I see is that this is a lot of work. Unless we can [automate it somehow like it's done in nixpkgs](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/servers/home-assistant/update.sh).

I started with the main requirements file https://github.com/home-assistant/core/blob/dev/requirements.txt and moved into the the constrained deps afterwards https://github.com/home-assistant/core/blob/dev/homeassistant/package_constraints.txt

Also a few that packages that have come up as missing modules during startup in the log.

I can see to reduce them to the minimum needed to get HQ started, if you prefer.

>     * What do you mean by "My newly added depenencies do not make use of the RDEPENDS handling"?

I meant RDEPENDS like this: https://github.com/meta-homeassistant/meta-homeassistant/blob/kirkstone/recipes-devtools/python/python3-hass-nabucasa_0.32.2.bb#L10

All of my newly added python recipes do not have any of this.

File: recipes-homeassistant/homeassistant/python3-homeassistant_2022.5.1.bb [hunk 1]
```diff
@@ -0,0 +1,126 @@
+SUMMARY = "Open-source home automation platform running on Python 3"
```
Comment (bachp) [inline recipes-homeassistant/homeassistant/python3-homeassistant_2022.5.1.bb:1]: We should directly update to 2022.5.3

**Review by bachp — CHANGES_REQUESTED** [review]
- Please remove the old version of recipes. The goal is not to have multiple versions of home assistant.

File: recipes-homeassistant/homeassistant/python3-homeassistant_2022.5.1.bb (hunk 1 above)
Comment (Stefan-Schmidt) [reply recipes-homeassistant/homeassistant/python3-homeassistant_2022.5.1.bb]: Updated to the latest version.

**Comment by Stefan-Schmidt** [comment]
I left the old recipes in to avoid regressions as I am not able to check on all use cases that people had with it.

Added a new commit that removes the old HQ recipes, as well as the dependencies which have been updated. I did that as a single extra commit to allow for easier revert when debugging a regression.

**Review by bachp — APPROVED** [review]

**Pull request merged**